models/POP/shell_scripts/cesm2_1/stitch_prior.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#
+# Copyright 2020 University Corporation for Atmospheric Research
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.
+# Please view the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# ==============================================================================
+
+#Aim: stitch together filter output to make diagnostic files
```

Comment:
This file is 99% the same as stich_post.sh ... can you make one script that takes an argument that would do the job of both files?